src/bun.js/webcore/blob.zig

```diff
@@ -1314,7 +1314,39 @@ pub const Blob = struct {
             return .undefined;
         };
 
-        const blob = Blob.findOrCreateFileFromPath(path, globalObject);
+        var blob = Blob.findOrCreateFileFromPath(path, globalObject);
+
+        if (arguments.len == 2) {
```

Comment:
```suggestion
        if (arguments.len >= 2) {
```

We should ignore extra arguments rather than require an exact count of arguments (since that's more common in JS)

Reply:
Fixed in https://github.com/oven-sh/bun/pull/6512/commits/775a31a2617aeae121824926e59c7e25fe733c2c